pandas/tests/frame/test_arithmetic.py

```diff
@@ -299,3 +299,18 @@ def test_df_bool_mul_int(self):
         result = 1 * df
         kinds = result.dtypes.apply(lambda x: x.kind)
         assert (kinds == 'i').all()
+
+    def test_td64_df_add_int_frame(self):
+        # Check that we don't dispatch to numpy implementation, which treats
+        # int64 as m8[ns]
+        tdi = pd.timedelta_range('1', periods=3)
+        df = tdi.to_frame()
+        other = pd.DataFrame([1, 2, 3], index=tdi)  # indexed like `df`
+        with pytest.raises(TypeError):
+            df + other
```

Comment:
Any relevant error message to check for?

Reply:
Not really.  It may be worth standardizing these once they all happen in one place, but at the moment its a hodge-podge.